### ISSUE TITLE
Prescribed Radiant Fraction not equivalent for each reaction in two-step scheme

### ISSUE THREAD

**Issue by rmcdermo** [firemodels/fds#10730]
I noticed that the second reaction in the two-step scheme is taking on the default radiant fraction of 0.35.  I don't think this is intended.  Look at the .out file for this simple methane case.  I think we want both reactions to have RN%CHI_R=0.2

```
&HEAD CHID='methane'/
&MESH IJK=4,4,4, XB=0,1,0,1,0,1/
&TIME T_END=.01/
&COMB N_SIMPLE_CHEMISTRY_REACTIONS=2/
&REAC FUEL='METHANE'/
&TAIL /
```

**Comment by rmcdermo** [comment]
Should be fixed with PR #10731

**Comment by mcgratta** [comment]
Yup, I was about to point out that line in read.f90
